### PR TITLE
Styling and Home Page completed

### PR DESCRIPTION
**Pull request by froheel**
I have updated the navigation bar and used css and .js (boot strap) to style the index.html page. Basically, homepage is completely designed with images, animations etc.

### CODE REVIEW

**Comment by froheel** [comment]
Completes Issues #37 and #34

File: src/main/webapp/css/index-page.css [hunk 1]
```diff
@@ -16,5 +16,50 @@
...
-	  background: url("../images/index-background.jpg");     
-}
\ No newline at end of file
+	  background: url("../images/index-1.jpg");
+}
+
+.container {
+    padding: 80px 120px;
+  }
+  .person {
```
Comment (waahm7) [inline src/main/webapp/css/index-page.css:25]: nit: try to keep the indentation consistent

**Review by waahm7 — APPROVED** [review]
Great work!

File: src/main/webapp/css/index-page.css (hunk 1 above)
Comment (froheel) [reply src/main/webapp/css/index-page.css:25]: Will keep that in mind. Thanks for the tip  :)

**Pull request merged**